static/docs/commands-reference/version.md

```diff
@@ -19,7 +19,7 @@ system/environment:
 | [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version)                                                                            |
 | `Python version`                            | Version of the Python being used for the project in which DVC is initialized                                                                              |
 | `Platform`                                  | Information about the operating system of the machine                                                                                                     |
-| [`Binary`](#what-we-mean-by-binary)         | Shows whether the package is installed from a binary release or source                                                                                    |
+| [`Binary`](#what-we-mean-by-binary)         | Shows whether the package is installed from a binary release or source code                                                                               |
```

Comment:
`or source code` - I would completely remove this part. It either "binary" or a package. Not source code.

Reply:
OK, updated in 19746b2

But in https://dvc.org/doc/commands-reference/version#what-we-mean-by-binary you can see it mentions installing from the Git repo, which wouldn't count as a package:

![image](https://user-images.githubusercontent.com/1477535/61500773-e4750a00-a989-11e9-9acc-1fbacc129271.png)

Reply:
@jorgeorpinel I think it's fine, since we explain it below. Github repo can is pip package anyway. Just a dev version of it.